test/data/stages/qemu/checks.json

```diff
@@ -29,6 +29,9 @@
   "image.vpc": {
     "format": "raw"
   },
+  "image-no-force-size.vpc": {
+    "format": "raw"
+  },
```

Comment:
So the way the test works is that it takes this dictionary and ensures that it is contained (a subset) of the `qemu-img info` output. But the reason to check only `format` for VPC in the past was that we were not allowing to explicitly set any other options so we didn't check anything else.

With the `force_size` set and not, it may make sense to also check the `virtual-size` value in the `qemu-img info` output. Otherwise, the test case does not test the actual effect of setting the new option.

Reply:
Good point :+1:  I was too much in a hurry to check if it actually works with RHEL7's qemu.

One thing though, I don't think the reasoning about _not_ checking the `virtual-size` for the "plain" case is actually right:

> But the reason to check only format for VPC in the past was that we were not allowing to explicitly set any other options so we didn't check anything else.

I think we want check the behaviour of the stage (specifically the one we depend on) and make sure we don't regress and thus since we set specific options we want to check for the effect of them (i.e. if force-size gets removed accidentally, resulting images don't import to Azure any more, but the test don't check this).  

Long story short, I will add the `virtual-size` to both.

Reply:
> One thing though, I don't think the reasoning about not checking the virtual-size for the "plain" case is actually right
I think that it was reasonable before this PR, although not ideal since the `force_size` was always set. I definitely agree that the best approach now is to check `virtual-size` in both cases 😉